start_cerberus.py

```diff
@@ -94,6 +97,11 @@ def main(cfg):
         while (int(iteration) < iterations):
             iteration += 1
 
+            for node in master_nodes:
+                taint = kubecli.get_taint_from_describe(node)
```

Comment:
This is specific to openshift, maybe add a high level config option called distribution ( default being openshift ) where users can set it to kubernetes or openshift and we can just run this check if it's openshift like @mffiedler suggested in https://github.com/openshift-scale/cerberus/issues/42. This way we can still run the subset of Cerberus checks on Kubernetes. Thoughts?

Reply:
I added this option because I do agree that it will be helpful to be able to run just kuberenetes checks vs kuberenetes and openshift. I set the default to openshift and only do this new check when the distribution is for openshift.

Reply:
Quick question, the check will also work on generic kubernetes right? We check the masters for node-role.kubernetes.io/master:NoSchedule taint ( I guess it will be empty for kubernetes in case the masters are schedulable by default ), so can we run this check irrespective of the distribution?

Reply:
Based on comments from Mike in the original issue, it is openshift specific

Reply:
Took the distribution option out and changed the oc describe to kubectl describe